File: src/Application/CustomerProperty/CustomerPropertyDto.cs
using AutoMapper;
using ReProServices.Application.Common.Mappings;
using System;

namespace ReProServices.Application.CustomerProperty

{
    public class CustomerPropertyDto : IMapFrom<Domain.Entities.CustomerProperty>
    {
        public int CustomerPropertyId { get; set; }
        public decimal? CustomerShare { get; set; }
        public int CustomerId { get; set; }
        public int PropertyId { get; set; }
        public DateTime DateOfSubmission { get; set; }
        public string UnitNo { get; set; }
        public string Remarks { get; set; }
        public bool IsShared { get; set; }
        public int? StatusTypeId { get; set; }
        public int? PaymentMethodId { get; set; }
        public int GstRateID { get; set; }
        public int TdsRateID { get; set; }
        public decimal? TotalUnitCost { get; set; }
        public bool? TdsCollectedBySeller { get; set; }
        public Guid? OwnershipID { get; set; }
        public string CustomerAlias { get; set; }
        public DateTime? DateOfAgreement { get; set; }
        public bool IsPrimaryOwner {get;set;}

        public bool? IsArchived { get; set; }
        public decimal? StampDuty { get; set; }
        public void Mapping(Profile profile)
        {
            profile.CreateMap<Domain.Entities.CustomerProperty, CustomerPropertyDto>();
        }


    }
}

File: src/WebUI/src/app/client/CustomerPropertyDto.ts
export class CustomerPropertyDto {
  public customerPropertyID: number;
  public customerShare: number;
  public customerID: number;
  public propertyID: number;
  public dateOfSubmission: Date;
  public unitNo: string;
  public remarks: string;
  public paymentMethod: number;
  public isShared: boolean;
  public statusTypeID: number;
  public paymentMethodID: number;
  public gSTRate: number;
  public tDSRate: number;
  public totalUnitCost: number;
  public agreementDate: Date;
  public allowForm16B: boolean;
  public tdsCollectedBySeller: boolean;
}
